llvm/test/DebugInfo/CAS/AArch64/debug_abbrev_split.ll

```diff
@@ -0,0 +1,51 @@
+; RUN: llc -O0 --filetype=obj --cas-backend --cas=%t.casdb --mccas-verify -o /dev/null %s
```

Comment:
os `/dev/null` something lit specifically knows about? I.e., would this test work on Windows?

Reply:
otherwise `%t.bin`  be a compatible alternative.

Reply:
Good catch!